framework/src/st_things/things_stack/easy-setup/easysetup_manager.c

```diff
@@ -224,7 +224,7 @@ esm_result_e esm_init_easysetup(int restart_flag, things_server_builder_s *serve
 		if (wifiIntf & 0x08) {
 			wifi_mode[mode_count++] = WiFi_11N;
 		}
-		if (wifiIntf & 0x10) {
+		if (wifiIntf & 0x16) {
```

Comment:
interfaces[int]: Supported Wi-Fi interfaces: b000001(a), b000010(b), b000100(g), b001000(n), or b010000(ac)

Reply:
b010000 = 0x10 
I think the old code is right.